ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

```diff
@@ -390,3 +390,50 @@ def test_paddle_margin_ranking_loss(
         margin=margin,
         reduction=reduction,
     )
+
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.multilabel_soft_margin_loss",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        shared_dtype=True,
+        exclude_min=True,
+        exclude_max=True,
+        min_num_dims=2,
+        min_dim_size=1,
+        max_dim_size=10,
+    ),
+    dtype_and_weight=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+    ),
+    reduction=st.sampled_from(["mean", "none", "sum"]),
+)
+def test_paddle_multilabel_soft_margin_loss(
+    dtype_and_x,
+    dtype_and_weight,
+    reduction,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    x_dtype, x = dtype_and_x
+    weight_dtype, weight = dtype_and_weight
+    helpers.test_frontend_function(
+        input_dtypes=[
```

Comment:
This parameter is supposed to be a list, not a nested list, you can concatenate both `x_dtype` and `weight_dtype` lists